### PR TITLE
RPZ add LC 11

### PR DESCRIPTION
**Pull request by mnaguit**
install Trillium 120QA
open LC 11

### CODE REVIEW

**Comment by mnaguit** [comment]
Changes done @samgeonet . thank you.

**Comment by ozym** [comment]
I think the title of the ticket is causing confusion, the sensor wasn't swapped (although that's probably the eventual outcome, we just don't have a borehole sensor to swap it with now).

**Comment by mnaguit** [comment]
Thanks @salichon & @ozym . I'll revise streams.csv.
The original ticket (https://github.com/GeoNet/tickets/issues/6442) says Guralp was removed. I assumed the new install in Trillium will stream LC 10 & new LC 11.

**Comment by mnaguit** [comment]
Hi @ozym . Do the changes in this PR look fine with you? thanks.

**Comment by mnaguit** [comment]
Also this one, please check @ozym . Thanks!

File: install/streams.csv [hunk 1]
```diff
@@ -1668,6 +1668,7 @@ RPCS,20,200,true,false,true,2002-06-24T00:00:00Z,9999-01-01T00:00:00Z
 RPZ,10,0.1,true,false,false,2001-06-06T00:00:00Z,9999-01-01T00:00:00Z
 RPZ,10,1,true,false,false,2001-06-06T00:00:00Z,9999-01-01T00:00:00Z
 RPZ,10,100,true,false,false,2001-06-06T00:00:00Z,9999-01-01T00:00:00Z
+RPZ,11,100,false,false,false,2020-05-01T07:34:00Z,9999-01-01T00:00:00Z
```
Comment (ozym) [inline install/streams.csv:1671]: Seems to be missing the 1 Hz and 0.1 Hz streams etc for the location 11.

Comment (mnaguit) [reply install/streams.csv:1673]: 1 & 0.1 hz streams now added. Thanks @ozym !

**Comment by mnaguit** [comment]
Hi @ozym @salichon @samgeonet 
I would just need one approval of review for this PR, and I'm ready to merge it. Thanks!

**Comment by mnaguit** [comment]
It looks like it's asking for 2 reviews for the recent changes.
@samgeonet & @ozym , can you please do another round of review for this? Thanks for your patience.

**Comment by mnaguit** [comment]
All good now. Ready to merge. Thanks @salichon !

**Pull request merged**